src/app/view/components/_common/action-header/action-header.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'action-header',
+    styles: [require('./action-header.scss')],
+    templateUrl: 'action-header.html'
+})
+
+export class ActionHeaderComponent {
+
+    public dynamicOverflow = 'visible';
+
+    @Input() selected = false;
+    @Input() selectedItems = [];
+    @Input() actionButtonItems = [];
```

Comment:
priming splitButton handle the MenuItem interface, so it should be something like:

@Input() actionButtonItems = MenuItem[];